Makefile

```diff
@@ -53,19 +55,26 @@ code/run/service_account: setup/service_account
 	$(MAKE) code/run
 
 .PHONY: code/compile
-code/compile:
+code/compile: code/gen
 	@GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o=$(COMPILE_TARGET) ./cmd/manager
 
+deploy/crds/integreatly.org_installations_crd.yaml: pkg/apis/integreatly/v1alpha1/installation_types.go
+	operator-sdk generate openapi
+pkg/apis/integreatly/v1alpha1/zz_generated.openapi.go: pkg/apis/integreatly/v1alpha1/installation_types.go
+	operator-sdk generate openapi
+
+pkg/apis/integreatly/v1alpha1/zz_generated.deepcopy.go:	pkg/apis/integreatly/v1alpha1/installation_types.go
+	operator-sdk generate k8s
+
 .PHONY: code/gen
-code/gen: deploy/crds/integreatly.org_installations_crd.yaml
+code/gen: setup/moq deploy/crds/integreatly.org_installations_crd.yaml pkg/apis/integreatly/v1alpha1/zz_generated.deepcopy.go pkg/apis/integreatly/v1alpha1/zz_generated.openapi.go
 	find ./ -name *_moq.go -type f -not -path "./vendor/*" -delete
-	operator-sdk generate k8s
-	operator-sdk generate openapi
 	@go generate ./...
 
 .PHONY: code/check
 code/check:
 	@diff -u <(echo -n) <(gofmt -d `find . -type f -name '*.go' -not -path "./vendor/*"`)
+	golint
```

Comment:
This is misleading, since it doesn't do anything. It should either be removed or changed to something that we care about like:

```bash
golint ./pkg/... ./cmd/... ./test/... ./version/... | grep -v "or be unexported" | grep -v "comment on exported"
```

Maybe leave it out for now, as it's not really relevant to this PR. WDYT?

Reply:
I agree, going to change it ASAP